src/main/webapp/i18n/en/complaint.json

```
@@ -69,6 +69,7 @@
             "info": "Submitting a feedback request does not affect your number of allowed complaints, but your score will remain unchanged.",
             "review": "Review More Feedback Request",
             "alreadyHandled": "The request has been resolved",
+            "alreadySubmitted": "You have already requested more feedback",
```

```suggestion
            "alreadySubmitted": "You already requested more feedback",
```
Can we simply use the here?